leveldb/src/main/java/org/iq80/leveldb/util/MergingIterator.java

```diff
@@ -95,90 +93,4 @@ public String toString()
         sb.append('}');
         return sb.toString();
     }
-
-    private static class ComparableIterator
```

Comment:
Now there no real benefit of moving this out of here, as you have removed the duplicated code.